text/src/autogluon/text/automm/data/process_text.py

```diff
@@ -35,6 +41,59 @@
 }
 
 
+def construct_augmenter(
+        augment_types: List[str],
+) -> naf.Sometimes:
```

Comment:
Actually you may change the return type to `Optional[naf.Sometimes]` since it can be None.